banzai_floyds/settings.py

```diff
@@ -5,15 +5,18 @@
     'banzai.trim.Trimmer',
     'banzai.gain.GainNormalizer',
     'banzai.uncertainty.PoissonInitializer',
+    'banzai.cosmic.CosmicRayDetector',
     'banzai_floyds.orders.OrderLoader',
     # Note that we currently don't apply the order tweak, only calculate it and save it in the header
     'banzai_floyds.orders.OrderTweaker',
     'banzai_floyds.wavelengths.WavelengthSolutionLoader',
     'banzai_floyds.fringe.FringeLoader',
     'banzai_floyds.fringe.FringeCorrector',
-    'banzai_floyds.extract.ProfileFitter',
+    'banzai_floyds.binning.Binner',
+    'banzai_floyds.profile.ProfileFitter',
+    'banzai_floyds.background.BackgroundFitter',
     'banzai_floyds.extract.Extractor',
-    'banzai_floyds.trim.Trimmer',
+    # 'banzai_floyds.trim.Trimmer',
```

Comment:
I was previously chopping off the ends of spectra. I think that was masking issues, so I've removed that for now.